packages/bp/src/common/utils.ts

```diff
@@ -17,3 +17,16 @@ export const sanitizeName = (text: string) =>
     .replace(/\s|\t|\n/g, '-')
     .toLowerCase()
     .replace(/[^a-z0-9-_.]/g, '')
+
+/**
+ *
+ * @param workspaceName string
+ * @param botId string
+ * @returns string that repects validation.ts BOT_ID_REGEX
+ */
+export const makeBotId = (workspaceName: string, botId: string) => {
```

Comment:
can we make the difference between a bot id and a bot id with a prepended workspace id? maybe `scopedBotID` ?

Reply:
also isn't the second parameter the `botName` ?